files/galaxy/tpv/tool_defaults.yml

```
@@ -42,6 +42,24 @@ tools:
               entity.tpv_tags = entity.tpv_tags.combine(
                   TagSetManager(tags=[pulsar_tag])
               )
+      - id: removed_remote_resources
+        # This rule displays a meaningful error message when users that have selected remote resources that are no longer available (e.g. because they have been removed) attempt to send jobs to them.
+        if: |
+          retval = False
+          remote_resource_tag = None
+          if user is not None:
+              try:
+                  user_preferences = user.extra_preferences
+                  remote_resource_tag = user_preferences.get("distributed_compute|remote_resources", "None")
```

```suggestion
                  remote_resource_tag = user_preferences.get("distributed_compute|remote_resources")
```

If you want you can omit that.